spec/book_spec.rb

```diff
@@ -0,0 +1,23 @@
+# Test for initialize method
+describe Book do
+  it 'should create instance of the book class with 2 parameters' do
+    book = Book.new('Title', 'Author')
+    expect(book).to be_an_instance_of Book
+  end
+end
+
+# Test for title
+describe '#title' do
+  it 'should return the right title' do
+    book = Book.new('My Title', 'My Author')
+    expect(book.title).to eq('My Title')
+  end
+end
+
+# Test for author
+describe '#author' do
+  it 'should return the right author' do
+    book = Book.new('My Title', 'My Author')
+    expect(book.author).to eq('My Author')
+  end
+end
```

Comment:
- [ ] Kindly import the Book class here as I was unable to run this test without importing it.

  ```
  require_relative '../book.rb'
  ```
  Or you can just import the `spec_helper.rb` file you defined.
  ```
  require_relative './spec_helper.rb'
  ```

  ![image](https://user-images.githubusercontent.com/54848363/203770119-5f3e10e1-ba62-4128-b1b7-ed9479ecdebe.png)

- [ ] All public methods should be tested. Kindly add tests for the `add_rental` method in the Book class.